README.md

```diff
@@ -14,6 +14,29 @@ Supports the following:
 - Writing a Pillow Image out as a HEIC file
 - Use Pillow functionality for other image manipulations - resize, transforms etc.
 
+Examples
+--------
+
+Import heif file into a PIL image
+
+```python
+from cyheifloader import cyheif
+
+img = cyheif.get_pil_image("my_heif_file.heic".encode()) # Filename must be in bytes
+# Now can use `img` as if you ran Image.open() from PIL or pillow 
+```
+
+View EXIF data
+
+```python 
+from cyheifloader import cyheif
+

```

Comment:
I believe you will also need:
`from PIL.ExifTags import TAGS`
for this example - thank you for adding these to the README